### DISCUSSION TITLE
How can we enable gh models in GitHub CLI?

### DISCUSSION THREAD

**Discussion by nafu** [community/community#136183] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Like this demo: https://www.youtube.com/watch?v=WiBB8Lsgl7I&t=110s

**Comment by hassanc** [comment]
Can't find any info or docs on this.. anyone figured out how to do this?

**Reply by Oleksii909** [reply]
Following the video i understand that i can use unix pipe operator to provide context for `gh models` command

**Reply by alexmazaltov** [reply]
I also cant find how to install the models extension for github cli. I tried `gh extension install github/gh-models` but it not works.

If you have any suggestions how to get `gh models .....` working. It would be appreciated.

**Comment by matthewisabel** [comment]
We'll be adding more info on the CLI extension into docs soon, sorry for the delay!

**Reply by hassanc** [reply]
Any updates on this?

**Comment by Akash1134** [comment]
Hi All, 

Thank you for your interest in using GitHub Models with the GitHub CLI. We appreciate your patience since this discussion began in August 2024.

## Current Status

We're pleased to announce that the GitHub CLI Models extension is now available for public use. This functionality has moved beyond the demo phase shown in the original video reference and is now properly documented and supported.

For comprehensive documentation on using GitHub Models in CLI, please visit:

- [GitHub Models CLI Extension Documentation](https://docs.github.com/en/github-cli/github-cli/creating-github-cli-extensions)
- [GitHub Models API Reference](https://docs.github.com/en/rest?apiVersion=2022-11-28)

We value your feedback on this new functionality. If you encounter any issues or have suggestions for improvements, please open a new discussion or contribute directly to the extension's repository.

We're marking this discussion as resolved, as the functionality is now fully available. Thank you for your interest and patience!
